resources/views/admin/dashboard.blade.php

```diff
@@ -0,0 +1,91 @@
+@extends('layouts.app')
+
+@section('content')
+
+{{-- temporaly use for front end coding--}}
+               <!-- Bootstrap -->
+               <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
```

Comment:
Since bootstrap is already in the project, we can remove this import of bootstrap